### ISSUE TITLE
Log shows a wrong name for plot images

### ISSUE THREAD

**Issue by simonech** [dotnet/BenchmarkDotNet#1181]
After running a benchmark with `[RPlotExporter]` I get zillions of jpg files, but I'm not sure all of them have the correct name.

First of all, the log in the console says:
```
// * Export *
  BenchmarkDotNet.Artifacts/results/MyBench.Sleeps-report.csv
  BenchmarkDotNet.Artifacts/results/MyBench.Sleeps-report-github.md
  BenchmarkDotNet.Artifacts/results/MyBench.Sleeps-report.html
  BenchmarkDotNet.Artifacts/results/MyBench.Sleeps-measurements.csv
  BuildPlots.R
  BenchmarkDotNet.Artifacts/results/MyBench.Sleeps-20190623-012647-boxplot.png
  BenchmarkDotNet.Artifacts/results/MyBench.Sleeps-20190623-012647-barplot.png
```
But the files stored in the artifacts folder are called `MyBench.Sleeps-barplot.png` and `MyBench.Sleeps-boxplot.png` (so, they don't have the timestamp).

**Comment by AndreyAkinshin** [comment]
@simonech thanks for the report!

**Comment by simonech** [comment]
@AndreyAkinshin I could have a look at the first, since it's an issue in the .net code.
The second is generated in the R script, and I've no idea how that works 😄

**Comment by AndreyAkinshin** [comment]
@simonech 
> I could have a look at the first, since it's an issue in the .net code.

Great, thanks! It should be pretty easy to fix.

> maybe it would be better to split the issue in two issues

OK. Do you want to create another issue for the second problem (and edit the current one)?

**Comment by simonech** [comment]
> @simonech
> 
> > I could have a look at the first, since it's an issue in the .net code.
> 
> Great, thanks! It should be pretty easy to fix.

Indeed... I'll do it

> 
> > maybe it would be better to split the issue in two issues
> 
> OK. Do you want to create another issue for the second problem (and edit the current one)?

Yeah, I'll create a new ticket with the second part of the issue

**Comment by AndreyAkinshin** [comment]
Great, thanks! Waiting for a PR. =)

**Comment by marcnet80** [comment]
Hi, @AndreyAkinshin. Could you please provide steps to reproduce this issue?
Maybe this behaviour relates to this line: https://github.com/dotnet/BenchmarkDotNet/blob/master/src/BenchmarkDotNet/Exporters/ExporterBase.cs#L34

**Comment by AndreyAkinshin** [comment]
@marcnet80 just run any benchmark with the `[RPlotExplorter]` attribute.
Since `RPlotExporter` generates a lot of different plots (not only `boxplot` and `barplot`) and there is no a simple way to get real names of these files (this logic is controlled by the internals of `BuildPlots.R`), I suggest to print just `*.png` instead of specific names.

**Comment by marcnet80** [comment]
@AndreyAkinshin, all generates plots have png extension?

**Comment by AndreyAkinshin** [comment]
@marcnet80 yes.

**Comment by marcnet80** [comment]
Ok. I will create pull-request today. Thank you.

**Comment by adamsitnik** [comment]
Fixed a long time ago in #1217, closing
